tests/doc_test/docs_needs_id_builder/conf.py

```diff
@@ -0,0 +1,162 @@
+#
```

Comment:
this file can be a lot more minimal; only include what is absolutely necessary to pass the tests